docs/guides/recovery/user-recovery.primary.in.rst

```diff
@@ -0,0 +1,67 @@
+Troubleshooting advice for RSP login issues.
+
+
+Problems with a new account
+===========================
+
+Attempts to login will not be successful until your account petition is approved by Rubin staff
+(i.e., step 8 of the process for :doc:`../getting-started/get-an-account`).
+This can take up to a week.
+Your patience is appreciated.
+
+Sometimes this approval process requires that Rubin staff verify that you have Rubin data rights.
+Please check your email's spam folder for a data rights verification
+request from `Heather Shaughnessy <mailto:sheather@slac.stanford.edu>`__.
+
+If you are at an institution outside the US or Chile, and you are a member of an international
+in-kind contribution team, but your name does not yet appear on the
+`list of international data rights holders <https://www.lsst.org/scientists/international-drh-list>`__,
+contact your program manager and Heather Shaughnessy to request to be added to the list.
+
+
+Being redirected to the new account setup process
+=================================================
+
+In cases where attempts to login are redirected into the process for setting up a new account,
+the issue is usually that the wrong identity provider was chosen at login.
+
+Try your other identities (e.g., GitHub, ORCID).
+
+After successfully logging in, link the identity you had tried to use so that it will work next time.
+See :doc:`../getting-started/linking-more-ids`.
+
+
+Forgotten password
+==================
+
+Rubin staff do not have access to your password or any other data from your identity provider besides your name.
+Follow up with your identity provider.
+
+
+Moved institutions (expired identity)
+=====================================
+
+If you no longer have access to any of the identity provider(s) associated with your
+RSP account, please request assistance.
+
+
+Requesting assistance
+=====================
+
+Please include in your request for account assistance:
+
+* if you have been able to log in before
+* any account details you remember (full name, email domain, identity provider, username)
+* whether your home directory contains needed data
+
+Regarding the last point,
+expunging an existing account and letting you start fresh can be the fastest and easiest option.
```

Comment:
Expunging an account does not delete the home directory previously used. It just deleted the CILogon record. This means that anybody who later picks the same username will get this user's files. This is a feature provided _you are certain you are dealing with the same user_ : you direct the user to use the same username (which you could even remind them of) when signing up after being expunged, which will return them to the same directory and their files. 

So it is critical that expunging happens immediately before the user re-signs up and with adequate confirmation of their identity.

Reply:
I just took out the whole thing about expunging. Heather et al. can make that decision on a case-by-case basis.